contracts/contracts/coordination/Coordinator.sol

```diff
@@ -876,4 +877,30 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
     //     require(tokenBalance > 0, "Insufficient balance");
     //     token.safeTransfer(msg.sender, tokenBalance);
     // }
+
+    function reorderParticipants(
+        uint32 ritualId,
+        uint256[][2] calldata pairs
```

Comment:
Actually, is this what you intended? Because Solidity expresses multidimensional arrays this in the reverse order.
https://docs.soliditylang.org/en/latest/types.html#arrays

Taking this into consideration, the test below is ambiguous...